Source/RomanNumeral.c

```diff
@@ -13,4 +13,24 @@ static char * Append(char *output, const char *toAppend)
 
 void RomanNumeral_Convert(unsigned input, char *output)
 {
+   char *CharactersArray []= {"M", "CM", "D", "CD", "C", "XC", "L", "XL", "X", "IX", "V", "IV", "I"};
+   unsigned DecimalArray []= {1000, 900, 500, 400, 100, 90, 50, 40, 10, 9, 5, 4, 1};
+   unsigned i, arrayAdress, tableSize;
+
+   for(i = 0; i <= 999; i++)
+      {
+         output[i] = 0;
+      }
```

Comment:
Two things here:
- the `i` is only used in this loop, let's create it here so its scope is delimited/defined.
- can you think of a different way of accomplish the same "reset to 0" effect but that is not restricted to N value? ~hint: string.h~